src/components/calcite-tree-item/calcite-tree-item.tsx

```diff
@@ -18,6 +18,10 @@ import { Scale } from "../interfaces";
 import { CSS, SLOTS, ICONS } from "./resources";
 import { CSS_UTILITY } from "../../utils/resources";
 
+/**
+ * @slot children - A slot for adding nested `calcite-tree` elements.
+ * @slot - A slot for adding content to the item.
```

Comment:
Sidebar: should we list the default slot first (general to specific)?

Reply:
Sure I can do that as part of #2974